packages/babel-helper-fixtures/src/index.js

```diff
@@ -1,6 +1,6 @@
-import cloneDeep from "lodash/cloneDeep";
-import clone from "lodash/clone";
-import extend from "lodash/extend";
+import cloneDeep from "lodash.clonedeep";
+import clone from "lodash.clone";
+import extend from "lodash.assignin";
```

Comment:
`Object.assign`?

Reply:
`lodash.extend` replaced with `Object.assign` here (and for one other remaining callsite) in https://github.com/babel/babel/pull/11812